internal/provider/auth_cert_test.go

```diff
@@ -317,9 +335,10 @@ func TestAuthLoginCert_Login(t *testing.T) {
 			handler: &testLoginHandler{
 				handlerFunc: handlerFunc,
 			},
-			token:     "foo",
-			wantErr:   true,
-			expectErr: errors.New("vault login client has a token set"),
+			token:      "foo",
+			cloneToken: true,
```

Comment:
The Auth Cert Login method clones the Vault client, and we want to ensure that the test client appropriately clones the token to result in this failing test case